microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -64,8 +64,16 @@ With this release, updates for both minor releases and patch releases are suppor
 //[id="microshift-4-14-support"]
 //=== Support
 
-//[id="microshift-4-14-post-installation"]
-//=== Post-installation configuration
+[id="microshift-4-14-post-installation"]
+=== Post-installation configuration
+
+[id="microshift-4-14-changes-config-yaml"]
+==== Configuration options with a custom YAML have changed
+* File placement for a custom YAML is now defined as `/etc/microshift/config.yaml`.
```

Comment:
This is not a new location for the configuration file. There used to be 2 possible names for the file. The change is that `$HOME/.microshift/config.yaml` is no longer used.